modules/va_facilities/README.yml

```diff
@@ -414,6 +414,161 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/APIError"
+  /nearby:
+    get:
+      tags:
+        - facilities
+      summary: Query facilities based on drive time from a specified point
+      description: |
+        Retrieve all facilities contained within an isochrone based on a the passed `lat`, `lng`, and `drivetime` parameters.
+
+        Additionally one can filter the facilities within the bounding box by type and available
+        services. Only facilities of type "health" and "benefits" may be filtered by available
+        services.
+
+        Results of this operation are paginated. JSON responses include pagination information
+        in the standard JSON API "links" and "meta" elements. GeoJSON responses include pagination
+        information in the "Link" header.
+      operationId: getNearbyFacilities
+      security:
+        - api_key: []
+      parameters:
+        - name: lat
+          in: query
+          description: |
+            Latitude of point to search for nearest VA Facilities.
+            Latitude and Longitude parameters should be specified in WGS84 coordinate reference system.
+          required: true
+          schema:
+            type: number
+            format: float
+        - name: lng
+          in: query
+          description: |
+            Longitude of point to search for nearest VA Facilities.
+            Latitude and Longitude parameters should be specified in WGS84 coordinate reference system.
+          required: true
+          style: form
+          schema:
+            type: number
+            format: float
+        - name: drivetime
+          description: Drive time is the maximum number of minutes to from the requested point that are included in the response.
```

Comment:
```suggestion
          description: Filter to only include facilities that are within the specified # of drive time minutes from the requested point.
```

Reply:
Yep, that is definitely clearer.

Reply:
Trying to figure out how to word this a little better. Does this suggestion work?